src/Arrays.php

```diff
@@ -445,4 +446,56 @@ public static function wrap(mixed $value, bool $wrapNull = true): array
 
         return [$value];
     }
+
+    /**
+     * Converts an array to a JSON string.
+     *
+     * @param array       $items
+     * @param string|null $failureMessage
+     *
+     * @return string
+     */
+    public static function toJson(array $items, ?string $failureMessage = null): string
+    {
+        if (count($items) === 0) {
+            throw new RuntimeException('Array is empty' . ($failureMessage === null ? '' : '. ' . $failureMessage));
+        }
+
+        try {
+            $json = json_encode($items, JSON_THROW_ON_ERROR);
+            if ($json === false) {
```

Comment:
json_encode doesn't return false when `JSON_THROW_ON_ERROR` is set. I assume phpstan issue?
just make this `return (string) json_encode(...)`